js/everything.js

```diff
@@ -59,6 +59,30 @@ function sendKeenEvent(_msg, _data) {
 var plyrPlayer;
 var youTubeDataApiKey = "AIzaSyCxVxsC5k46b8I-CLXlF3cZHjpiqP_myVk";
 var currentVideoID;
+var currentListID;
+var currentListIndex;
+var listData;
+var isPlayingPlaylist = false;
+
+function getSource(url) {
+        var xmlHttp = null;
+        xmlHttp = new XMLHttpRequest();
+        xmlHttp.open( "GET", url, false );
+        xmlHttp.send( null );
+        return xmlHttp.responseText;
+}
+
+function loadList(listID) {
```

Comment:
This is necessarily complicated, let's move this logic to `zap-common.js` and use the same format as the jQuery HTTP requests.